coverage/.resultset.json

```diff
@@ -1,49 +1,135 @@
 {
```

Comment:
Thinking we should add this folder to .gitignore as it keeps changing across our PCs.

coverage/